includes/class-wc-payments-account.php

```diff
@@ -1317,6 +1318,7 @@ public function maybe_handle_onboarding() {
 				}
 
 				$this->cleanup_on_account_reset();
+				delete_transient( self::WOOPAY_TEST_DRIVE_SETTINGS_FOR_LIVE_ACCOUNT );
```

Comment:
why not move it inside `$this->cleanup_on_account_reset();`? 🙂 it's called in a few places

Reply:
the method is called also for when switching to live account, and we don't want to remove it at that point

Reply:
that makes sense